google_guest_agent/diagnostics_integ_test.go

```diff
@@ -0,0 +1,15 @@
+//go:build integration
+// +build integration
+
+package main
+
+import (
+	"testing"
+)
+
+func TestDiagnosticsManager(t *testing.T) {
+	var d = diagnosticsMgr{}
+	if !d.disabled("linux") {
+		t.Errorf("linux system does not support diagnose")
```

Comment:
all we do is validate the disabled function? i don't think that's adding much value

Reply:
i agree that this test is probably needed for linux as diagnosticsManager is only available on windows